homeassistant/components/esphome/strings.json

```diff
@@ -6,7 +6,7 @@
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     },
     "error": {
-      "resolve_error": "Can't resolve address of the ESP. If this error persists, please set a static IP address: https://esphomelib.com/esphomeyaml/components/wifi.html#manual-ips",
+      "resolve_error": "Can't resolve address of the ESP. If this error persists, please set a static IP address: https://esphome.io/components/wifi.html#manual-ips",
```

Comment:
I guess we need to decide on this. We can either accept this; or, decide to adjust the wording and not use an URL at all.

What should we do? (I lean towards the latter).

Reply:
I removed the link in the error text.